app/models/backup.rb

```diff
@@ -1,9 +1,13 @@
-class Backup < Export
+class Backup < ApplicationRecord
   class << self
-    def permission
+    def create_permission
       :create_backup
     end
 
+    def restore_permission
+      :restore_backup
+    end
+
     def include_attachments?
       val = OpenProject::Configuration.backup_include_attachments
```

Comment:
I think it is ok not to include the NextCloud files but yes, restoring a backup might mess things up there. I think that's unavoidable, though. It might lose the connection to NextCloud altogether because it hadn't been configured yet at the time of the backup or perhaps the credentials have changed in the meantime.

@Kharonus Can you think of something on the NextCloud side that could break when one restores a backup? I guess it could be that work packages referenced in files no longer exist. Can the OpenProject extension in NextCloud handle that?

I reckon it can. I think this is just something one has to accept when doing this.